### PR TITLE
Add reference link for full-text search

### PR DESCRIPTION
**Pull request by estherk15**
### What does this PR do? What is the motivation?

Hyperlink logs full-text search to the wikipedia definition of what a full-text search is.
[DOCS-8388](https://datadoghq.atlassian.net/browse/DOCS-8388)

### Merge instructions

Do not merge

### Additional notes

[DOCS-8388]: https://datadoghq.atlassian.net/browse/DOCS-8388?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: content/en/logs/explorer/search.md [hunk 1]
```diff
@@ -19,15 +19,15 @@ further_reading:
 
 While information from individual logs can be useful visualized as a list, sometimes valuable information can be accessed through aggregation. To access this information, search for logs in the [Log Explorer][5] and display them as timeseries, top lists, tree maps, pie charts, or tables.
 
-Log Explorer search consists of a time range and a search query, mixing `key:value` and full-text search. 
+Log Explorer search consists of a time range and a search query, mixing `key:value` and [full-text search][6].
```
Comment (estherk15) [inline content/en/logs/explorer/search.md:22]:
```suggestion
Log Explorer search consists of a time range and a search query, mixing `key:value` and [full-text search][6].  **Note**: Full-text search supports searching texts in logs.  It utilizes n-gram and morphemic analysis. If precision is required, search with `key:value`.
```

**Review by maycmlee — APPROVED** [review]

**Comment by usman-dd** [comment]
I'm not convinced that "it utilizes n-gram and morphemic analysis" is the type of information our customers are looking for in search docs. If users are confused about what full-text search means, we should aim to make things simpler and clarify, not add more esoteric terms that the average user will have to look up.

**Comment by estherk15** [comment]
> I'm not convinced that "it utilizes n-gram and morphemic analysis" is the type of information our customers are looking for in search docs. If users are confused about what full-text search means, we should aim to make things simpler and clarify, not add more esoteric terms that the average user will have to look up.

Thanks @usman-dd, is this something that might warrant a separate guide or explanation? I don't have context on whether this is a common misunderstanding. If not, I can recommend an internal KB.

File: content/en/logs/explorer/search.md [hunk 2]
```diff
@@ -94,3 +94,4 @@ You can interact with the search bar with your mouse, as well as by using keyboa
 [3]: /logs/search-syntax
 [4]: /dashboards/guide/custom_time_frames
 [5]: /logs/explorer/
+[6]: https://en.wikipedia.org/wiki/Full-text_search
```
Comment (estherk15) [inline content/en/logs/explorer/search.md:97]:
```suggestion
[6]: /logs/explorer/search_syntax/#full-text-search
```

File: content/en/logs/explorer/search.md [hunk 3]
```diff
@@ -19,15 +19,15 @@ further_reading:
 
 While information from individual logs can be useful visualized as a list, sometimes valuable information can be accessed through aggregation. To access this information, search for logs in the [Log Explorer][5] and display them as timeseries, top lists, tree maps, pie charts, or tables.
 
-Log Explorer search consists of a time range and a search query, mixing `key:value` and full-text search. 
+Log Explorer search consists of a time range and a search query, mixing `key:value` and [full-text search][6].  **Note**: Full-text search supports searching texts in logs.  It utilizes n-gram and morphemic analysis. If precision is required, search with `key:value`.
```
Comment (estherk15) [inline content/en/logs/explorer/search.md:22]:
```suggestion
Log Explorer search consists of a time range and a search query, mixing `key:value` and [full-text search][6].
```

**Pull request merged**